Activities/ManagePersonsPermissionCheckerTask.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.Extensions.Localization;
+using OrchardCore.Security.Services;
+using OrchardCore.Users.Models;
+using OrchardCore.Users.Services;
+using OrchardCore.Workflows.Abstractions.Models;
+using OrchardCore.Workflows.Activities;
+using OrchardCore.Workflows.Models;
+using OrchardCore.Workflows.Services;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Lombiq.TrainingDemo.Activities;
+
+// A simple workflow activity task that accepts an username as a TextField input and checks whether the user has
+// ManagePersons Permission or not.
+public class ManagePersonsPermissionCheckerTask : TaskActivity
+{
+    private readonly IStringLocalizer S;
+
+    private readonly IWorkflowExpressionEvaluator _expressionEvaluator;
+
+    private readonly IUserService _userService;
+
+    private readonly IRoleService _roleService;
+
+
+    public ManagePersonsPermissionCheckerTask(
+        IUserService userService,
+        IWorkflowExpressionEvaluator expressionEvaluator,
+        IRoleService roleService,
+        IStringLocalizer<ManagePersonsPermissionCheckerTask> localizer)
+    {
+        _expressionEvaluator = expressionEvaluator;
+        S = localizer;
+        _userService = userService;
+        _roleService = roleService;
+    }
+
+    public override string Name => nameof(ManagePersonsPermissionCheckerTask);
+
+    public override LocalizedString DisplayText => S["Manage Persons Permission Checker Task"];
+
+    public override LocalizedString Category => S["User"];
+    public WorkflowExpression<string> UserName
```

Comment:
```suggestion
    public override LocalizedString Category => S["User"];
    
    public WorkflowExpression<string> UserName
```

Reply:
If you accept, apply these suggestions directly so the reviewer doesn't need to eyeball them.